src/rosdep2/sources_list.py

```diff
@@ -493,12 +496,18 @@ def update_sources_list(sources_list_dir=None, sources_cache_dir=None,
         print('Add distro "%s"' % dist_name)
         rds = RosDistroSource(dist_name)
         rosdep_data = get_gbprepo_as_rosdep_data(dist_name)
+        # Store Python version from REP153
+        if distribution.get('python_version'):
+            python_versions[dist_name] = distribution.get('python_version')
```

Comment:
For this to be available `rosdep` should declare a version dependency on the upcoming `rosdistro` version 0.7.5.

Reply:
Bumped dependency on `rosdistro` to `0.7.5` in f32222f